tests/timezones/tests.py

```diff
@@ -51,6 +61,13 @@
 EAT = timezone.get_fixed_timezone(180)      # Africa/Nairobi
 ICT = timezone.get_fixed_timezone(420)      # Asia/Bangkok
 
+PARIS_IMPLS = [
```

Comment:
Hey @pganssle — I'm being slow (as ever 🙂) what's `IMPL` stand for here? (Ta!)

Reply:
Yeah, implementation. I could also change this to be something more like I did in the other tests, a `def get_timezones(key)` function that returns a list of the specific implementations rather than using a global like this.

Reply:
I went ahead and made that change in a fixup commit so you can see what it looks like. Let me know what you want to do and I'll either squash it down or drop the new commit.